README.md

```diff
@@ -14,40 +32,52 @@ Using libsir, you can make a single call that simultaneously sends information t
 
 Each 'level' or 'priority' of output can be visually styled however you wish for `stdio` destinations. This allows for immediate visual confirmation that a message appeared in the terminal that you should pay attention to, while less pressing information is more subtle in appearance.
 
-![libsir visual graph](./docs/res/libsir-alpha.gif)
+![libsir visual graph](./docs/res/libsir-alpha.png)
```

Comment:
The gif is much smaller; any reason to change to png?

Reply:
I did it because the GIF tries to play as a video in a lot of browsers, like Firefox, and instead of being resized with a nice algo, it's resized as a fast "thumbnail" ...

New: ![Screenshot from 2023-07-08 23-16-26](https://github.com/aremmell/libsir/assets/61629094/1392ded0-c0fc-4507-a3a1-2b2e72402072)

Current: ![Screenshot from 2023-07-08 23-17-07](https://github.com/aremmell/libsir/assets/61629094/bfa620a9-6a4b-4f13-a58a-2025a76a3b31)

Notice the difference in initial quality and the weird play button (that does nothing.)